forc/Cargo.toml

```
@@ -41,6 +41,7 @@ toml_edit = "0.13"
 ureq = { version = "2.4", features = ["json"] }
 url = "2.2"
 uwuify = { version = "^0.2", optional = true }
+walkdir = "2"
```

Should we use major.minor here?
```suggestion
walkdir = "2.3"
```